armotypes/portaltypesutils.go

```diff
@@ -31,53 +31,58 @@ func AttributesDesignatorsFromWLID(wlid string) *PortalDesignator {
 }
 
 func (designator *PortalDesignator) GetCluster() string {
-	cluster, _, _, _, _ := designator.DigestPortalDesignator()
+	cluster, _, _, _, _, _ := designator.DigestPortalDesignator()
 	return cluster
 }
 
 func (designator *PortalDesignator) GetNamespace() string {
-	_, namespace, _, _, _ := designator.DigestPortalDesignator()
+	_, namespace, _, _, _, _ := designator.DigestPortalDesignator()
 	return namespace
 }
 
 func (designator *PortalDesignator) GetKind() string {
-	_, _, kind, _, _ := designator.DigestPortalDesignator()
+	_, _, kind, _, _, _ := designator.DigestPortalDesignator()
 	return kind
 }
 
 func (designator *PortalDesignator) GetName() string {
-	_, _, _, name, _ := designator.DigestPortalDesignator()
+	_, _, _, name, _, _ := designator.DigestPortalDesignator()
 	return name
 }
+func (designator *PortalDesignator) GetPath() string {
+	_, _, _, _, path, _ := designator.DigestPortalDesignator()
+	return path
+}
 func (designator *PortalDesignator) GetLabels() map[string]string {
-	_, _, _, _, labels := designator.DigestPortalDesignator()
+	_, _, _, _, _, labels := designator.DigestPortalDesignator()
 	return labels
 }
 
 // DigestPortalDesignator - get cluster namespace and labels from designator
-func (designator *PortalDesignator) DigestPortalDesignator() (string, string, string, string, map[string]string) {
+func (designator *PortalDesignator) DigestPortalDesignator() (string, string, string, string, string, map[string]string) {
```

Comment:
I think we should have a struct named `designator` or something like that, this struct will be returned from this function.
I see that you added here another field, but I think it's time to write it correctly